docs/blog/write-the-docs-2020-stats.rst

```diff
@@ -0,0 +1,224 @@
+.. post:: Jan 12, 2021
+   :tags: stats, year in review, newsletter
+   :author: Eric Holscher
+
+Write the Docs 2020 Stats and Community Update
+==============================================
+
+A new year comes with many expectations.
+Hoping for change,
+setting new goals,
+and feeling optimistic about the fresh year that has just begun.
+
+Sadly this year that optimism has been dashed in record time.
+We have written many times throughout 2020 that we hope you're okay,
+and we will have to continue that tradition once more in our inaugural newsletter of 2021.
+
+We hope you're okay,
+and we hope that the storm we have been engulfed in will soon pass.
+
+A heartfelt thanks
+------------------
+
+We had a lot of big plans for this year,
+but most of those had to be put on hold in March as we scrambled to move our community fully online.
+Getting to see all the happy faces at our in-person events is one of the largest motivators for people who organize events.
+Sadly this year, we weren't able to make that happen.
+
+2020 has shown that the love and care we put into the community will be returned back to us when needed.
+When we moved the conferences online,
+we were genuinely worried about the health and longevity of Write the Docs.
+As we noted in our `ticket choices`_ page for Portland,
+we had a lot of outstanding debts and we didn't know how much support we would get with our online conferences.
+It was easy to see a worst case scenario where our venues didn't refund us and people didn't support the online events.
+
+We are happy to report that we received overwhelming support from our vendors, sponsors, speakers, team, and attendees:
+
+* Most of our vendors refunded our down payments in full
```

Comment:
Maybe we can add that our live-captions and video broadcasting vendors followed us seamlessly to the virtual format